Dockerfile

```
@@ -0,0 +1,16 @@
+FROM aiidateam/aiida-prerequisites:latest
+
+USER root
+
+# Copy and install AiiDA
+COPY . aiida-core
```

isn't this the other way around?
```suggestion
COPY aiida-core .
```

---

no, I am copying the current folder `.` to the destination folder `aiida-core`

---

Right. Do we want the full git repo in the docker image?
The .git folder is about 100MB... but maybe you are right and we should have it. Hm.

---

To me, for the moment it doesn't harm. If we discover problems, then look at this issue in more details.